### ISSUE TITLE
keep search functionality for name (EAC-CPF) records

### ISSUE THREAD

**Issue by regineheberlein** [pulibrary/pulfalight#294]
copying over from #120:
faithc:

```
I'd like to add here that we would like the name (eac-cpf) records to have the same search functionality that they do on pulfa 2.0: https://findingaids.princeton.edu/names.

Also, @regineheberlein is currently working on updating these records. (We didn't think we'd be able to do so before the MVP was released, but with the changing timeline and such, this has been able to happen.) We/ @regineheberlein intend to import some records into the hosted ArchivesSpace instance once that is up and running so that staff have a chance to review the records that are exported from there.
```

**Comment by regineheberlein** [comment]
This may exceed the scope of the MVP, but I'm also wondering whether the `@resourceRelationType` could be used to generate a label indicating the nature of the related link, e.g.
```
<resourceRelation xlink:type="simple"
				xlink:href="http://findingaids.princeton.edu/collections/C0108/c3368"
				resourceRelationType="subjectOf" xlink:title="Hemingway, Ernest"/>
```
(This is not currently implemented.)

**Comment by jrgriffiniii** [comment]
Definitely, this is reasonable to support given that we are already extracting strings from the XML for indexing into Solr Documents for PULFALight.

**Comment by jrgriffiniii** [comment]
I will need to review this further, my suspicion is that this is going to require a nested Solr Document for modeling individual `name` entities. An alternative strategy might be to implement an ActiveRecord model and provide some Solr-compliant mapping strategy for the model attributes, but this is slightly more labor intensive.

**Comment by jrgriffiniii** [comment]
I've attempted this before with https://github.com/pulibrary/pulfalight/pull/257, and the approach wasn't too terrible to implement.

**Comment by jrgriffiniii** [comment]
This will require wireframing for an early attempt to implement the Show View.

**Comment by jrgriffiniii** [comment]
It is desired in order to ensure that name authorities will be retrieved with keyword searches, however disambiguation is going to be needed for the search result layout design: https://xd.adobe.com/view/a54bb217-10ff-43b2-72f6-6c3cadf376a4-d37a/screen/3a6835a5-8d84-4e46-9425-143716210f0e

**Comment by jrgriffiniii** [comment]
The enhanced UI updates for the search results view might not all be possible to implement for the initial MVP release, and this could be scoped for a later minor release.

**Comment by tpendragon** [comment]
See https://github.com/pulibrary/pulfalight/issues/309 for some notes.
